nncf/binarization/algo.py

```diff
@@ -71,7 +71,9 @@ def _binarize_weights_and_module_inputs(self, target_model: NNCFNetwork) -> List
                 op_weights = self.__create_binarize_module().to(device)
 
                 nncf_logger.info("Adding Activation binarizer in scope: {}".format(scope_str))
-                op_inputs = UpdateInputs(ActivationBinarizationScaleThreshold(module.weight.shape)).to(device)
+                compression_lr_scale = self.config.get("compression_lr_scale", None)
```

Comment:
Should call it `compression_lr_multiplier`, as we already have `scale` referring to the quantizer scale.

Reply:
Done.